### DISCUSSION TITLE
Inquiry Regarding Student Verification Status

### DISCUSSION THREAD

**Discussion by Nasangjoon** [community/community#138271] [GitHub Education]
### Hi everyone,

Subject: Inquiry Regarding Student Verification Status

Dear GitHub Support Team,

I hope this message finds you well.

I am writing to inquire about the status of my student verification. It has been five days since my verification was approved, but I am still unable to access the student benefits.

Could you please provide me with an update on when I can expect to gain access? If there are any additional steps I need to take or further information required from my end, please let me know.

Thank you very much for your assistance.

**Comment by davevad93** [comment]
Hi @Nasangjoon ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Nasangjoon was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
